md-rest-api/src/main/java/com/wipro/ats/bdre/md/rest/ExecStatusAPI.java

```diff
@@ -66,11 +67,10 @@ RestWrapper get(
                 execStatus.setExecStateId(jpaExecStatus.getExecStateId());
                 execStatus.setDescription(jpaExecStatus.getDescription());
             }
-            // execStatus = s.selectOne("call_procedures.GetExecStatus", execStatus);
-
             restWrapper = new RestWrapper(execStatus, RestWrapper.OK);
             LOGGER.info("Record with ID:" + execStateId + " selected from ExecStatus by User:" + principal.getName());
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "Record with ID:" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1192)